core/src/main/java/org/apache/servicecomb/core/CseApplicationListener.java

```diff
@@ -71,6 +72,8 @@ public void onApplicationEvent(ApplicationEvent event) {
       }
 
       SCBEngine.getInstance().init();
+    } else if (event instanceof ContextClosedEvent) {
+      SCBEngine.getInstance().uninit();
```

Comment:
I think it's better to use destroy, any ideas？

Reply:
done
changed all uninit to destroy